packages/teleport/src/components/LabelSelector/LabelSelector.tsx

```diff
@@ -0,0 +1,269 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+import Text from 'design/Text';
+import { Info, Warning } from '../../../../design/src/Icon';
+import Pill from '../../../../design/src/Pill';
```

Comment:
```suggestion
import { Info, Warning } from 'design/Icon';
import Pill from 'design/Pill';
```

Reply:
you can also export Pill and Popover in `design/src/index.js` and you can do combine imports

`import {Text, Box, Pill, Popover} from 'design'`